doc/advanced-topics.md

```diff
@@ -102,6 +103,27 @@ accepted. Provided the `type` is provisioned correctly, the `value` may be defin
 }
 ```
 
+Because GeoJSON types (e.g. `Point`, `LineString` etc.) are native types in **NGSI-LD**, automatic GeoJSON conversion is switched on for NGSI-LD by default.
+
+With **NGSI-v2**, for backwards compatibility reasons, automatic GeoJSON conversion for types other than `geo:json` is turned off by default.
+Add the `autocastGeoJSON` configuration to the attribute to enable GeoJSON conversion. Each GeoJSON attribute can be provisioned as shown:
+
+```json
+{
+    "entity_type": "GPS",
+    "resource":    "/iot/d",
+    "protocol":    "PDI-IoTA-JSON",
+..etc
+    "attributes": [
+        {
+            "name": "observationSpace",
+            "type": "Polygon",
+            "autocastGeoJSON" : "true"
+        }
```

Comment:
I think the following syntax would be better:

```
        {
            "name": "observationSpace",
            "type": "Polygon",
            "autocast" : true
        }
```

So the way in which the autocast is done depends on the `type`. This way we can extend the functionality in the future to autocast other types. E.g. to autocast to boolean (e.g. `"true"` -> `true`, `0` -> `false`):


```
        {
            "name": "observationSpaceEnabled",
            "type": "Boolean",
            "autocast" : true
        }
```

Maybe it would even better to have a `autocastType` to cover cases in which the NGSI type need to be customized by the user, eg:

```
        {
            "name": "observationSpace",
            "type": "MyCustomType",
            "autocastType": "Polygon",
            "autocast" : true
        }
```

The `autocastType` could be optional and, in the case it is ommited, `type` is used.